third_party/rust-hypervisor-firmware-virtio/src/pci.rs

```diff
@@ -391,7 +391,7 @@ impl VirtioTransport for VirtioPciTransport {
     }
 
     fn set_used_ring(&self, addr: u64) {
-        // queue_used: 0x28
+        // queue_used: 0x30
```

Comment:
For this comment: it doesn't give me much more insight, so I'd think we could either we expand or remove it.

Reply:
This is part of imported third_party code. I just fixed the typo which caused some confusion.